docs/troubleshooting/troubleshooting.md

```diff
@@ -0,0 +1,112 @@
+# Troubleshooting
+
+Sometimes you hit a roadblock. That's natural, in coding.
+
+Below are a few problems our users have seen, and ways they solved them in the
+past. If what you're looking for is not on this page, let us know at
+[help@streamlit.io](mailto:help@streamlit.io).
+
+## First things to try...
+
+We try to fix bugs quickly, so many times a problem will go away when you
+upgrade Streamlit. So the first thing to try when having an issue is upgrading
+to the latest version of Streamlit:
+
+```bash
+$ pip install --upgrade streamlit
+$ streamlit version
+```
+
+...and then verify that the version number printed is `0.50.2`.
```

Comment:
Is the version number here hard coded or will it update when we push a new version?